src/main/java/com/plaid/client/request/InstitutionsGetRequest.java

```diff
@@ -20,4 +22,21 @@ public InstitutionsGetRequest(Integer count, Integer offset) {
     this.count = count;
     this.offset = offset;
   }
+
+  public InstitutionsGetRequest(Integer count, Integer offset, boolean includeInstitutionData) {
```

Comment:
can we add a .withIncludeInstitutionData(boolean) instead of creating more constructors?

Reply:
yes will do